FFMediaToolkit/Decoding/Internal/InputContainer.cs

```diff
@@ -156,30 +140,29 @@ private static InputContainer MakeContainer(string url, MediaOptions options, AV
         /// <param name="options">The <see cref="MediaOptions"/> object.</param>
         private void OpenStreams(MediaOptions options)
         {
-            if (options.StreamsToLoad == MediaMode.AudioVideo || options.StreamsToLoad == MediaMode.Video)
-                Video = DecoderFactory.OpenVideo(this, options);
-
-            if (options.StreamsToLoad == MediaMode.AudioVideo || options.StreamsToLoad == MediaMode.Audio)
-                Audio = DecoderFactory.OpenAudio(this, options);
```

Comment:
I think it would be nice to preserve the functionality to enable video-only or audio-only reading. Do you think it's easy enough to include - e.g. here in OpenStreams you could set null in Decoders for streams that are of not-requested type and GetPacketFromStream you could skip buffering the packets for them

Reply:
I agree, this can be done quite easily simply by setting the decoders to null as you are saying.  I'll make sure to add this back in.